src/components/Footer/Footer.stories.js

```diff
@@ -1,88 +1,87 @@
+import React from "react";
 import { Footer } from "./Footer";
-import logo from "../../../public/wmms-blk.svg";
+import { DateModified } from "../DateModified/DateModified";
+import { ActionButton } from "../ActionButton/ActionButton";
+import logo from "../../assets/wmms-blk.svg";
+import { ReportAProblem } from "../ReportAProblem/ReportAProblem";
 
 export default {
-  title: "In Development/Footer",
+  title: "Components/Footer",
   component: Footer,
 };
 
 const Template = (args) => <Footer {...args} />;
 
 export const Primary = Template.bind({});
 Primary.args = {
```

Comment:
Would it make sense to have any other stories here for the footer (eg. possibly one with just the required fields and then one with all fields)?